src/main/java/me/totalfreedom/totalfreedommod/blocking/PlayerBlocker.java

```diff
@@ -0,0 +1,44 @@
+package me.totalfreedom.totalfreedommod.blocking;
+
+import com.google.common.collect.Lists;
+import me.totalfreedom.totalfreedommod.FreedomService;
+import me.totalfreedom.totalfreedommod.TotalFreedomMod;
+import me.totalfreedom.totalfreedommod.config.ConfigEntry;
+import me.totalfreedom.totalfreedommod.util.FLog;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.block.BlockPlaceEvent;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.Collection;
+import java.util.List;
+
+public class PlayerBlocker extends FreedomService
+{
+
+    public static final List<String> blockedTags = Lists.newArrayList();
+
+    public PlayerBlocker(TotalFreedomMod plugin)
+    {
+        super(plugin);
+    }
+
+    @Override
+    protected void onStart()
+    {
+        // Load banned tags
+        blockedTags.clear();
+        blockedTags.addAll((Collection<? extends String>) ConfigEntry.BLOCKED_TAGS.getList());
```

Comment:
You don't need a separate service for loading this list. Especially since we're never adding to it. You can simply use `ConfigEntry.BLOCKED_TAGS.getList()` anywhere you need it. Lookup should be pretty fast and without I/O.

Reply:
Just going to go and facepalm. Apparently I decided over-engineering it was the best way. Thanks for the heads up, I'll make appropriate changes.